pkg/cli/update.go

```diff
@@ -0,0 +1,151 @@
+package cli
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"syscall"
+
+	"github.com/google/go-github/github"
+	"github.com/sosedoff/pgweb/pkg/command"
+)
+
+var (
+	latestVersion string
+	workDir       = "/tmp"
+	distName      = fmt.Sprintf("pgweb_%s_%s", runtime.GOOS, runtime.GOARCH)
+	zipName       = distName + ".zip"
+)
+
+func checkUpdate() {
+
+	if options.DisableCheckUpdate {
+		return
+	}
+
+	client := github.NewClient(nil)
+
+	latestRelease, _, err := client.Repositories.GetLatestRelease(context.Background(), "sosedoff", "pgweb")
+	if err != nil {
+		fmt.Errorf("GitHub is unavailable")
+	}
+
+	latestVersion = *latestRelease.Name
```

Comment:
You can use `latestRelease.GetName()` instead

Reply:
Fixed